src/css/plot_styles.css

```diff
@@ -321,20 +321,16 @@ div.d3chart svg .Cement[class*='Integrated facility'] {
 	fill: #c1b358;
 }
 
-div.d3chart svg .Steel[class*='Ac-Electric Arc Furnace'] {
-	fill: #a63d57;
+div.d3chart svg .Steel[class*='Electric Arc Furnace'] {
+	fill: #dbb1bb;
 }
```

Comment:
NB (and not so related to this PR):
We should probably add logging in case the technology types input to the dashboard are not considered in `plot_styles.css`. I don't have a better way of doing this in mind, but can imagine that it might lead to some odd bugs in case data isn't as expected, and at the very least we should make it easy for ourselves to debug

Reply:
Can probably just make that comment into a new issue.

Reply:
I added an issue